webpack.config.js

```diff
@@ -10,5 +10,17 @@ module.exports = {
     globalObject: "this"
   },
   mode: "production",
-  devtool: "source-map"
+  devtool: "source-map",
+    optimization: {
```

Comment:
First we have to make indentations here consistent (it's over-indented now), and then you'll soon find what's wrong 🤣

Reply:
😂😂 okay I will revise it, let me see what's wrong in it

Reply:
```suggestion
  optimization: {
```

This is indeed over-indented, though 😉